lang/en/plagiarism_urkund.php

```diff
@@ -66,6 +67,16 @@
 $string['idisagree'] = 'Cancel';
 $string['id'] = 'ID';
 $string['identifier'] = 'Identifier';
+$string['setmahara'] = 'Set the Mahara URL, and the HTML Lite OAuth secret and key here to check portfolio submissions.';
+$string['maharawshtmlliteenabled'] = 'Check Mahara portfolios';
+$string['maharawshtmllitekey'] = 'Mahara HTML Lite OAuth key';
+$string['maharawshtmllitekey_help'] = 'The HTML Lite OAuth key from the partner Mahara sute.';
```

Comment:
"sute" - > "site" ? :-) - few other examples below... I'd also be tempted to remove the word "Partner" from those and just make it "the Mahara site"